src/Debugger/Impl/DataInspection/RValueProperties.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.R.DataInspection;
+using Microsoft.R.StackTracing;
+using static System.FormattableString;
+
+namespace Microsoft.R.DataInspection {
+    /// <summary>
+    /// Used to specify properties of <see cref="IRValueInfo"/> to fill when evaluating expressions using
+    /// <see cref="RSessionExtensions.EvaluateAndDescribeAsync"/>,
+    /// <see cref="RStackFrameExtensions.EvaluateAndDescribeAsync"/>,
+    /// or <see cref="REvaluationInfoExtensions.DescribeChildrenAsync"/>.
+    /// </summary>
+    [Flags]
+    public enum RValueProperties : ulong {
```

Comment:
I really don't like this name. It's technically correct in a sense that all members here _currently_ correspond to properties of `IRValueInfo`. But 1) this may not necessarily be true in the future, and 2) it is actually passed to various methods that return `IREvaluationInfo`, which may or may not be `IRValueInfo`, which is confusing.

I considered renaming it to `REvaluationProperties`, but that also feels weird, because it reads as "properties of evaluation", not "properties of what is produced by evaluation".

I would really like to get the names right in this refactoring, to avoid sweeping changes across all the code that uses it in the future, and this is a particularly high-traffic type. So, I would really appreciate any suggestions on the naming.